index.html

```diff
@@ -1,42 +1,161 @@
+            </ul>
+            <p class="lang-descr">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="lang-list">
+                <li class="lang-list-descr">html</li>
+                <li  class="lang-list-descr">css</li>
+                <li  class="lang-list-descr">javaScript</li>
+            </ul>
+            <button class="btn-project">See Project</button>
+        </div>
+        </div>
+    </section> 
+
+          <section class="about-myself">
+            <div class="div-about">
+            <h2 class="about-header">About Myself</h2>  
+            <p class="self-descr">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                  
+                       <a class="connect-page">LET'S CONNECT</a>
+                   
+                   <ul class="div-list">
+                       <li class="list-item"><img src="images/tweeter.png" alt="follow me on twitter" width="20" height="20" ></li>
+                       <li class="list-item"><img src="images/linkedin.png" alt="follow me on linkedin" width="20" height="20" ></li>
+                       <li class="list-item"><img src="images/medium.png" alt="follow me on medium" width="20" height="20" ></li>
+                       <li class="list-item"><img src="images/github.png" alt="follow me on github" width="20" height="20" ></li>
+                       <li class="list-item"><img src="images/angelist.png" alt="follow me on angellist" width="20" height="20" ></li>
+                   </ul>
+                   <button class="btn-resume">Get my resume</button>
+                   <div class="flex-container">
+                    <div class="flex-item">Languages <img src="images/Union.png" ></div>
+                       <div class="grid-container">
+                        <div class="grid-item"><image src="images/js.png" alt="jsicon" width="50" height="50"></div>
+                            <div><p class="grid-template-text">JavaScript</p></div>
+                        
+                         <div class="grid-item"><image src="images/html.png" alt="html" width="50" height="51"></div>
+                             <div><p class="grid-template-text">HTML</p></div>
+                        
+                          <div class="grid-item"><image src="images/css.png" alt="css" width="50" height="51"></div>
+                                 <div><p class="grid-template-text">CSS</p></div>
+                       </div>
+                    
+                     
+                    <div class="flex-item">Frameworks <img src="images/right.png"></div>
+                    <div class="flex-item">Skills <img src="images/right.png"></div>
+                   
```

Comment:
- [ ] Please make sure that your image in this section are showing :blush: 
![Screenshot from 2023-01-12 17-42-03](https://user-images.githubusercontent.com/92303891/212140582-1be1e6cc-85d8-4427-bedc-9607ea91dd7d.png)